sdk/appconfiguration/app-configuration/package.json

```diff
@@ -74,12 +74,12 @@
     "@azure/core-asynciterator-polyfill": "^1.0.0",
     "@azure/core-http": "^1.0.0",
     "@azure/core-paging": "^1.1.0",
-    "@azure/core-tracing": "1.0.0-preview.7",
-    "@opentelemetry/types": "^0.2.0",
+    "@azure/core-tracing": "1.0.0-preview.8",
+    "@opentelemetry/api": "^0.5.1",
     "tslib": "^1.10.0"
   },
   "devDependencies": {
-    "@azure/identity": "1.1.0-preview1",
+    "@azure/identity": "1.1.0-preview.2",
```

Comment:
I recall there being another PR that incremented the version for identity...
cc @KarishmaGhiya

Reply:
This change is required. I had only updated so that the exact version preview1 was pinned. See PR-  https://github.com/Azure/azure-sdk-for-js/pull/7753/files 
I updated to preview1 instead of preview.2 that time bcz all the packages were not released yet.
Was suppose to send a follow up PR to update to preview.2 (but slipped out of my mind)

Reply:
@KarishmaGhiya Can you send the follow up PR now? I'd like this PR to only have changes related to tracing

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/8069

Reply:
#8069 is merged, time for another merge from master!

Reply:
@ramya-rao-a updated!